modules/common/services/audio.nix

```diff
@@ -44,7 +44,7 @@ in
                 # Enable TCP socket for VMs pulseaudio clients
                 "server.address" = [
                   {
-                    address = "tcp:${toString cfg.pulseaudioTcpPort}";
+                    address = "tcp:0.0.0.0:${toString cfg.pulseaudioTcpPort}";
```

Comment:
It listens on all interfaces by default, but adding 0.0.0.0 disables IPv6. Is that intentional?

Reply:
Yes disabling ipv6 was intentional.
I did not know that we need ipv6 for pulseaudio, i can remove the change if it is needed.
Not that we currently do any firewalling for pulseaudio but to me it seems all of our firewalling is done in ipv4 so everyting on ipv6 is wide open.